ts/util/attachmentsUtil.ts

```diff
@@ -387,18 +387,21 @@ export const saveAttachmentToDisk = async ({
   messageTimestamp,
   messageSender,
   conversationId,
+  index,
 }: {
   attachment: AttachmentType;
   messageTimestamp: number;
   messageSender: string;
   conversationId: string;
+  index: number;
```

Comment:
can you make the `  index?: number;` not optional in the function `export const save = ({` in the file `ts/types/Attachment.ts` too please?

Reply:
I can, but this function is called at other places which require the default value to be calculated.